trade-finance-manager-api/src/v1/controllers/user/helpers/payload.js

```diff
@@ -29,6 +29,16 @@ const payloadVerification = (payload, template) => {
     return payloadKeyDataType.toLowerCase() === templateKeyDataType.toLowerCase();
   });
 
+  if(!propertiesExists) {
+    console.error('ERROR no required property', propertiesExists);
+  }
+  if(!noAdditionalProperties) {
+    console.error('ERROR extra properties', noAdditionalProperties);
+  }
+  if(!propertiesDataTypeMatch) {
+    console.error('Error property type is wrong', propertiesDataTypeMatch);
+  }
```

Comment:
Can we add some more context to the logs, if I saw these when debugging I wouldn't know what has the error.

Should these also throw an error?